tests/deci_core_unit_test_suite_runner.py

```diff
@@ -143,6 +144,7 @@ def _add_modules_to_unit_tests_suite(self):
         self.unit_tests_suite.addTest(self.test_loader.loadTestsFromModule(DeprecationsUnitTest))
         self.unit_tests_suite.addTest(self.test_loader.loadTestsFromModule(TestMinSamplesSingleNode))
         self.unit_tests_suite.addTest(self.test_loader.loadTestsFromModule(TestSegmentationMetricsMultipleIgnored))
+        self.unit_tests_suite.addTest(self.test_loader.loadTestsFromModule(TrainWithTorchSchedulerTest))
```

Comment:
This is great we are adding tests cases. 💪 
One test that I really really want to see here - test cases for all pytorch LR schedulers that we declare support. Not only this would give us some peace of mind that all of them are working properly but also can act as an usage examples.

Reply:
Second wish that came to my mind after looking through the PR - we should test torch schedules on regular models (subclassing from nn.Module) and on our ones with custom LR per layer (that has this `update_param_groups` thingie)

Reply:
Added testss for all of the torch schedulers in aa6e2347f8afc0793084e1488958510611c67d29

Regarding the second point - I think torch schedulers should not work (Ill add a warning) alongside update_param_groups.